sda-commons-server-jackson/build.gradle

```diff
@@ -7,11 +7,5 @@ dependencies {
 
   testImplementation project(':sda-commons-server-testing')
   testImplementation project(':sda-commons-shared-forms')
-  testImplementation 'com.kjetland:mbknor-jackson-jsonschema_2.13', {
-    /**
-     * Dropwizard comes with jakarta.validation-api instead of javax.validation-api.
-     * Both contain the same classes.
-     */
-    exclude group: 'javax.validation', module: 'validation-api'
-  }
+  testImplementation "io.github.classgraph:classgraph"
```

Comment:
NIT: I looked up, why we need Classgraph here and found a very picky detail.

The [variable here](https://github.com/SDA-SE/sda-dropwizard-commons/blob/39c053d/sda-commons-server-jackson/src/test/java/org/sdase/commons/server/jackson/errors/JerseyValidationExceptionMapperTest.java#L46) is prefixed with `javax` but references a `jakarta` package.

Reply:
Eagle eye 🦅 😆 
Fixed it!